apps/dashboard/src/@/components/in-app-wallet-users-content/searchUsers.ts

```diff
@@ -1,7 +1,79 @@
+import type {
+  ListUserWalletsData,
+  ListUserWalletsResponses,
+} from "@thirdweb-dev/api";
+import { configure, listUserWallets } from "@thirdweb-dev/api";
```

Comment:
btw i realize that we havent updated this package in a while, will do it after this for you

EDIT: nevermind i see you did it already